jekyll/assets/js/main.js

```diff
@@ -18,4 +18,10 @@ $( document ).ready(function() {
 	$("article h2, article h3, article h4, article h5, article h6").filter("[id]").hover(function () {
 		$(this).find("i").toggle();
 	});
+
+	$.getJSON("/api/v1/me").done(function (userData) {
+		$(document.body).addClass('loggedin');
+		analytics.identify(userData['analytics_id']);
+		window.userData = userData;
```

Comment:
do we need this part? I know it was copied over from cci.com, but if we don't need it lets remove it